app/services/api/get_trainees_service.rb

```diff
@@ -2,14 +2,14 @@
 
 module Api
   class GetTraineesService < BaseService
-    def initialize(provider:, params: {})
+    def initialize(provider:, params: {}, version:)
```

Comment:
If these services are going to be namespaced under `Api`, shouldn't they also be versioned? So instead of passing a version in here, you'd have `Api::GetTraineesService::V01` or `Api::V01::GetTraineesService` if we decide to swap them around.

Reply:
I'm not sure will it be premature to do that.

Currently I see the service in two veins
1. `Common`, that should work regardless of `version`
2. `Versioned`, that should only work for that `version`

So taken the path of least resistance and injected as a parameter.

Once it becomes clearer when we work on `V01` we can further define the namespacing

Reply:
Yeah that makes sense, happy to stick with this for now then 👍